src/PaymentMethods/AbstractPaymentMethod.php

```diff
@@ -15,21 +14,26 @@ abstract class AbstractPaymentMethod implements PaymentMethodI
     /**
      * @var string
      */
-    public $id;
+    protected $id;
```

Comment:
Why change this? :thinking: 

It seems like above it was used outside of the class.

https://github.com/mollie/WooCommerce/blob/dc46b46f5a0a1d41adae31ffaf39446ce5e421b5/src/Payment/PaymentModule.php#L496-L497